mixin/README.md

```diff
@@ -0,0 +1,30 @@
+# Go Runtime Mixin
+
+_This mixin is a work in progress. We aim for it to become a good role model for
+dashboards eventually, but it's not there yet._
+
+Mixins are a collection of configurable, reusable Prometheus rules, alerts
+and/or Grafana dashboards for a particular system, usually created by experts
+in that system. By [applying them]() to Prometheus and Grafana, you can quickly
```

Comment:
Link missing.

Reply:
whoops!

Reply:
i think i'm actually just gonna remove this for now - eventually i want that to be a link, but the docs to which i would link aren't really written yet